packages.yml

```diff
@@ -3,4 +3,5 @@ packages:
     version: [">=0.6.0", "<0.7.0"]
 
   - git: https://github.com/fivetran/dbt_fivetran_utils.git
+    revision: enabled_vars_one_true   #To remove after sales receipt test
```

Comment:
I've approved the PR in fivetran_utils so this can be pinned back to master.

Reply:
Updated to only install master branch